knowledge-base/kb-numericupdown-howto-introduce-event-fired-when-repeatbuttons-are-released.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Introduce Event That is Fired When RepeatButtons are Released
```

Comment:
I think "Create" (instead of "Introduce") would be more appropriate because it is a more common word.